docs/source/ProgrammingGuide/DeprecatedInitialization.md

```diff
@@ -0,0 +1,117 @@
+Kokkos chooses the two spaces using the following list:
+
+1. `Kokkos::Cuda`
+2. `Kokkos::Experimental::OpenMPTarget`
+3. `Kokkos::Experimental::HIP`
+4. `Kokkos::Experimental::SYCL`
+5. `Kokkos::OpenMP`
+6. `Kokkos::Threads`
+7. `Kokkos::Experimental::HPX`
+8. `Kokkos::Serial`
+
+The highest execution space in the list which is enabled is Kokkos' default execution space, and the highest enabled host execution space is Kokkos' default host execution space. For example, if  `Kokkos::Cuda`, `Kokkos::OpenMP`, and `Kokkos::Serial` are enabled, then `Kokkos::Cuda` is the default execution space and `Kokkos::OpenMP` is the default host execution space.<sup>1</sup>  In cases where the highest enabled backend is a host parallel execution space the `DefaultExecutionSpace` and the `DefaultHostExecutionSpace` will be the same.
+
+Command-line arguments come in "prefixed" and "non-prefixed" versions. Prefixed versions start with the string `--kokkos-`. [`Kokkos::initialize`](../API/core/initialize_finalize/initialize) will remove prefixed options from the input list, but will preserve non-prefixed options. Argument options are given with an equals (`=`) sign. If the same argument occurs more than once, the last one is used. Furthermore, prefixed versions of the command line arguments take precedence over the non-prefixed ones. For example, the arguments
+
+    --kokkos-threads=4 --threads=2
+
+set the number of threads to 4, while
+
+    --kokkos-threads=4 --threads=2 --kokkos-threads=3
+
+set the number of threads to 3. Table 5.1 gives a full list of command-line options.
+
+<h4>Table 15.1: Command-line options for Kokkos::initialize</h4>
+
+Argument | Description
+:---      | :---
+--kokkos-help  <br/> --help   | print this message
+--kokkos-threads=INT <br/> --threads=INT  | specify total number of threads or number of threads per NUMA region if used in conjunction with `--numa` option.
+--kokkos-numa=INT <br/> --numa=INT | specify number of NUMA regions used by process.
+--device-id=INT | specify device id to be used by Kokkos.
+--num-devices=INT[,INT] | used when running MPI jobs. Specify the number of devices per node to be used. Process to device mapping happens by obtaining the local MPI rank and assigning devices round-robin. The optional second argument allows for an existing device to be ignored. This is most useful on workstations with multiple GPUs, one of which is used to drive screen output.
+
+***
+<sup>1</sup> This is the preferred set of defaults when CUDA and OpenMP are enabled. If you use a thread-parallel host execution space, we prefer Kokkos' OpenMP back-end, as this ensures compatibility of Kokkos' threads with the application's direct use of OpenMP threads. Kokkos cannot promise that its Threads back-end will not conflict with the application's direct use of operating system threads.
+
```

Comment:
In the case of such a conflict, will it become readily apparent that this is happening?  If not, should it be made apparent?  Do such conflicts impact program correctness and performance?  Whatever the situation, if the Kokkos does not flag such issues, should it?